src/Generating/Trees.cpp

```diff
@@ -947,29 +988,35 @@ void GetJungleTreeImage(int a_BlockX, int a_BlockY, int a_BlockZ, cNoise & a_Noi
 
 void GetLargeJungleTreeImage(int a_BlockX, int a_BlockY, int a_BlockZ, cNoise & a_Noise, int a_Seq, sSetBlockVector & a_LogBlocks, sSetBlockVector & a_OtherBlocks)
 {
-	// TODO: Generate proper jungle trees with branches
-
-	// Vines are around the BigO4, but not in the corners; need proper meta for direction
-	static const sMetaCoords Vines[] =
+	static const sMetaCoords VinesTrunk[] =
 	{
-		{-2, -5, 1}, {-1, -5, 1}, {0, -5, 1}, {1, -5, 1}, {2, -5, 1},  // North face
-		{-2,  5, 4}, {-1,  5, 4}, {0,  5, 4}, {1,  5, 4}, {2,  5, 4},  // South face
-		{5,  -2, 2}, {5,  -1, 2}, {5,  0, 2}, {5,  1, 2}, {5,  2, 2},  // East face
-		{-5, -2, 8}, {-5, -1, 8}, {-5, 0, 8}, {-5, 1, 8}, {-5, 2, 8},  // West face
-		// TODO: vines around the trunk, proper metas and height
-	} ;
```

Comment:
Why remove the vines around the leaves?

Reply:
Vanilla doesn't have them either. Also, I didn't really like the look of all the long vines hanging from the leaves.

Reply:
Weird, I believed it did have them, dunno why. Alright then,